templates/index.html

```diff
@@ -34,6 +34,7 @@
   {% include "takeovers/_french_takeover_openstack_made_easy.html" %}
   {% include "takeovers/_french_14-04-esm.html" %}
   {# ALL #}
+  {% include "takeovers/_1904-takeover.html" %}
```

Comment:
@solazio and @anthonydillon today, this will be the only takeover until next tuesday, we should just comment out the rest